packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -692,6 +692,10 @@ async function determineBundler(
       name: 'webpack',
       message: 'Webpack [ https://webpack.js.org/ ]',
     },
+    {
+      name: 'rspack',
```

Comment:
I think this option should be added in this file too: https://github.com/nrwl/nx/blob/master/packages/create-nx-workspace/bin/types/bundler-list.ts